_posts/2020-07-15-#19031.md

```diff
@@ -12,17 +12,22 @@ commit:
 
 ## Notes
 
-* [PR 19031](https://github.com/bitcoin/bitcoin/pull/19031) is the
-  implementation of [BIP
-  155](https://github.com/bitcoin/bips/blob/master/bip-0155.mediawiki), the
-  `addrv2` message.
+* [PR 19031](https://github.com/bitcoin/bitcoin/pull/19031) is a proposed implementation of the [BIP 155](https://github.com/bitcoin/bips/blob/9286b5254317d9e73fb25c5f0acd2b2d9937843e/bip-0155.mediawiki) `addrv2` messages, a new p2p message format proposed in early 2019 by Wladimir J. van der Laan to gossip longer node addresses. This is required to support new-generation Onion addresses, I2P, and potentially other networks that have longer endpoint addresses than fit in the 128 bits of the current addr message.
 
-* The smaller PR that's currently being worked on, and should be reviewed
-  first, is [PR 19360](https://github.com/bitcoin/bitcoin/pull/19360).
+* The proposed implementation in PR 19031 was turned in a roadmap and separated into a series of smaller PRs to ease review. The first one in the series, [PR 19351](https://github.com/bitcoin/bitcoin/pull/19351) "test: add an edge case test for CSubNet", has been merged. [PR 19360](https://github.com/bitcoin/bitcoin/pull/19360) "net: improve encapsulation of CNetAddr" is the second PR in the series and currently under review.
```

Comment:
Oops, we write "proposed implementation" at the start of both first two paragraphs. Suggestion:

"To ease review, PR 19031 was turned into a roadmap and separated into a series of smaller PRs."